README.md

```diff
@@ -12,19 +12,19 @@
 <p align="center">Describe, connect, secure, test and deploy distributed applications with less effort.</p>
 
 <p align="center">
-    <a href="https://godoc.org/github.com/inspr/inspr">
-        <img src="https://godoc.org/github.com/inspr/inspr?status.svg">
+    <a href="https://godoc.org/inspr.dev/inspr">
+        <img src="https://godoc.org/inspr.dev/inspr?status.svg">
     </a>
-    <a href="https://goreportcard.com/badge/github.com/inspr/inspr">
-        <img src="https://goreportcard.com/badge/github.com/inspr/inspr">
+    <a href="https://goreportcard.com/badge/inspr.dev/inspr">
+        <img src="https://goreportcard.com/badge/inspr.dev/inspr">
     </a>
-    <a href="https://pkg.go.dev/github.com/inspr/inspr">
+    <a href="https://pkg.go.dev/inspr.dev/inspr">
         <img src="https://img.shields.io/static/v1?label=godoc&message=reference&color=blue">
     </a>
     <a href="https://codecov.io/gh/inspr/inspr">
         <img src="https://codecov.io/gh/inspr/inspr/branch/develop/graph/badge.svg?token=C8SPTHPXMG&">
     </a>
-    <a href="https://github.com/inspr/inspr/blob/develop/LICENSE.md">
+    <a href="https://inspr.dev/inspr/blob/develop/LICENSE.md">
```

Comment:
@nicholaspcr This is a link to the license on GitHub, so doesn't go through vanity